src/main/java/com/sforce/cd/apexUnit/client/utils/ApexClassFetcherUtils.java

```diff
@@ -432,5 +433,24 @@ public static void logTheFetchedApexClasses(String[] classIds) {
 			}
 		}
 	}
+	
+	/*
+	 * Returns a string array of namespace and class/trigger name
+	 * 
+	 * @param strLine - String value of user provided <NAMESPACE>.<CLASS/TRIGGER>
+	 */
+	public static Map<String, String> parseNamespaceAndName(String strLine){
```

Comment:
Yes @adarsh-ramakrishna-sfdc , actually we use this class [here](https://github.com/forcedotcom/ApexUnit/blob/namespace-to-query/src/main/java/com/sforce/cd/apexUnit/client/fileReader/ApexManifestFileReader.java#L124)

Actually in this class it is not used currently, but the idea is to use it when namespace and name are parsed for regex as well.